content-src/components/Sections/_Sections.scss

```diff
@@ -16,13 +16,17 @@
     }
 
     @media (min-width: $break-point-large) and (max-width: $break-point-large + 2 * $card-width) {
-      :nth-child(3n) {
+      // 3n for normal cards, 4n for compact cards
+      :nth-child(3n),
+      :nth-child(4n) {
```

Comment:
I think we  don't need this one? Or maybe it's the one below... Not sure:

<img width="846" alt="screen shot 2018-10-11 at 11 15 50 am" src="https://user-images.githubusercontent.com/36629/46814948-d7b2ee00-cd47-11e8-9994-843f774abf2a.png">

Reply:
removing this one seems to still work, i wasn't sure if it was for something else so i just left it in there for safety. I can take it out though

Reply:
From what I can tell, this `@media {}` block is for when the browser is wide enough to show 3 compact columns but not 4. Sort of that in between state. So only the `3n` should be needed.